openapi/types.rs

```diff
@@ -82,7 +82,16 @@ pub struct FlightOption {
     pub timestamp_depart: SystemTime,
 
     /// Estimated arrival timestamp
-    pub timestamp_arrive: SystemTime
+    pub timestamp_arrive: SystemTime,
+
+    /// The estimated trip distance
+    pub distance_km: f32,
+    
+    /// The currency type, e.g. USD, EUR
+    pub currency_type: Option<String>,
+
+    /// The cost of the trip for the customer
+    pub customer_cost: Option<f32>
```

Comment:
Maybe call this `base_pricing`? This is the upfront cost calculated by `svc-pricing`. And if we do layer in auction later on, we would have an additional field called `final_pricing` or `accepted_pricing`.

Reply:
https://github.com/Arrow-air/svc-cargo/pull/14/commits/5bfa8c58e96525ccf1913bdf1e3ff75f85a5c8e8